dist/jquery.mark.d.ts

```diff
@@ -47,12 +46,16 @@ declare namespace Mark {
     shadowDOM?: boolean | ShadowObject;
 
     filter?(
-      textNode: Text, term: string, totalMatchesSoFar: number, termMatchesSoFar: number, filterInfo: MarkFilterInfo
```

Comment:
Not sure why it seemed to change the formatting on build... didn't really change anything in that regard 🤔